cli/src/SRGAutomation/evaluate/SRGCommandEvaluate.ts

```diff
@@ -68,9 +69,16 @@ class SRGCommandEvaluate implements BaseCommand {
       .default("90")
       .env("SRG_EVALUATION_DELAY");
 
+    const multipleSRG = new Option(
+      "--multiple-guardians [multipleGuardians]",
```

Comment:
neither "--multiple-guardians" nor "SRG_MULTIPLE_GUARDIANS=true" works for multiple SRG evaluation